gradle/platform/build.gradle.kts

```diff
@@ -1,5 +1,5 @@
 plugins {
-    id("java-platform")
+    `java-platform`
```

Comment:
Thanks for suggesting this, but I don't want the changes in the commit for three reasons:
- I would like the scripts to look consistent. It's confusing for new users that you sometimes us `id(...)` and sometimes it works without
- It does not work in Groovy DSL, I want these script to work there as well wherever possible
- In Gradle 8, it's actually faster to only used `id`: https://docs.gradle.org/8.0-rc-3/release-notes.html#kotlin-dsl

Reply:
Good to know.